src/createAsyncThunk.ts

```diff
@@ -392,7 +399,7 @@ If you want to use the AbortController to react to \`abort\` events, please cons
     arg: ThunkArg
   ): AsyncThunkAction<Returned, ThunkArg, ThunkApiConfig> {
     return (dispatch, getState, extra) => {
-      const requestId = nanoid()
+      const requestId = (options?.idGenerator || nanoid)()
```

Comment:
:hand: We can use nullish coalescing here instead of `||`, right?

Reply:
How exactly are you thinking? As in `options?.idGenerator?.() || nanoid()`?

Reply:
`(options?.idGenerator ?? nanoid)()`

just swapping `||` for `??`. same behavior, slightly more semantically correct.

Reply:
Sorry just realised you said nullish coalescing (??). Definitely makes more sense, will do

Reply:
Now addressed